kogito-pmml-quickstart/src/test/java/org/kie/pmml/kogito/quarkus/LogisticRegressionIrisDataTest.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.pmml.kogito.quarkus;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.http.ContentType;
+import org.hamcrest.core.IsNull;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+import java.util.Map;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.*;
+
+@QuarkusTest
+class LogisticRegressionIrisDataTest {
+
+    private static final String BASE_PATH = "/LogisticRegressionIrisData";
+    private static final String DESCRIPTIVE_PATH = BASE_PATH + "/descriptive";
+    private static final String TARGET = "Species";
+    private static final String INPUT_DATA = "{\n" +
+            "  \"Sepal.Length\": 6.9,\n" +
+            "  \"Sepal.Width\": 3.1,\n" +
+            "  \"Petal.Length\": 5.1,\n" +
+            "  \"Petal.Width\": 2.3\n" +
+            "}";
+
+    @Test
+    void testEvaluateLogisticRegressionIrisDataResult() {
+        given()
+                .contentType(ContentType.JSON)
+                .body(INPUT_DATA)
+                .when()
+                .post(BASE_PATH)
+                .then()
+                .statusCode(200)
+                .body(TARGET, is(52.5f));
```

Comment:
AFAICS, this is incorrect and should be `"virginica"`. I fixed it and I'm running the native tests to see how it goes. I also upgraded the PR to 1.9.0.

Reply:
Thanks @gsmet 
There were  also an issue about start imports - I hope it is fixed now